curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/step-010.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 616d47bc9eedc4bc7f621bec
+title: Step 10
+challengeType: 0
+dashedName: step-10
+---
+
+# --description--
+
+Next, within the `div`, add another `div` element and give it a class of `marker`.
+
+# --hints--
+
+Your new `div` element should have an opening tag.
+
+```js
+assert([...code.matchAll(/<div.*?[^>]>/gi)][1]);
```

Comment:
Currently, this only matches:
```
<divt>
```
Because, after `div`, you are searching for something that has to be between the `v` and the `>`

Lazy matching should be enough

```suggestion
assert([...code.matchAll(/<div.*?>/gi)][1]);
```

Reply:
I committed that 😬 - hopefully nobody is in the middle of working on this still